### DISCUSSION TITLE
CSS themes for fomantic

### DISCUSSION THREAD

**Discussion by manuelpoelzl** [fomantic/Fomantic-UI#494] [General] [answered]
# Help Wanted

## Problem
Hi!
I was just wondering if there is something like the semantic-ui theme forest for fomantic-ui.
For example css only themes, which allows me to use the raised theme without the need of less.

Thanks in advance

**Comment by y0hami** [comment, marked as answer]
@manuelpoelzl Currently no there isn't, if you want to use the raised theme you could however build it and then just copy the dist into your project. Then you wouldn't need to include the LESS code in your project.

**Comment by manuelpoelzl** [comment]
@hammy2899  Thanks for your answer, i will do it this way!
